types/qs/qs-tests.ts

```diff
@@ -10,7 +10,7 @@ qs.parse('a=b&c=d', { delimiter: '&' });
 () => {
     let obj = qs.parse('a=z&b[c]=z&d=z&d=z&e[][f]=z');
     obj; // $ExpectType ParsedQs
-    obj.a; // $ExpectType string | ParsedQs | string[] | ParsedQs[]
+    obj.a; // $ExpectType string | ParsedQs | string[] | ParsedQs[] | undefined
```

Comment:
Maybe add a test to cover your specific use case, so it's clearer why we want to include `undefined`?

Reply:
Something like:

```ts
declare const myQuery: { a: string; b?: string }
const myQueryCopy: ParsedQs = myQuery;
```

Reply:
I added the interface extension test that should cover the same thing, but I can also add this one no problem.

Reply:
Added in df7ec738a73d787712d9b71013720c513a29486b

Reply:
I have already PR to add undefined to [ParsedQS](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/44617#pullrequestreview-408767023)

Reply:
Ah, sorry, completely missed it. I can close this one then, maybe you can just port over the couple of test cases I added to cover the use case.

Reply:
No problem. It's completely fine. I have add your test case into the PR #44617 with this commit 7c85f0aa0127835782d67ebfcf2a86d6979a0638

Reply:
Perfect, thanks, then I'll close this one.